src/Tribe/Editor/Attendee_Registration.php

```diff
@@ -87,7 +87,12 @@ public function admin_init() {
 	 * @return string success message
 	 */
 	public function success_notice() {
-		return '<div class="success"><p>' . __( 'Attendee Registration fields saved.', 'event-tickets' ) . '</p></div>';
+		$link    = '<a href="' . get_edit_post_link( $this->post->ID, 'raw' ) . '">' . __( 'return to the content editor', 'event-tickets' ) . '</a>';
+		$notice  = '<div class="success"><p>';
+		$notice .= sprintf( __( 'Attendee Registration fields saved. Make additional changes or %1$s', 'event-tickets' ), $link );
```

Comment:
Should have escaping

```suggestion
		$notice .= sprintf( esc_html__( 'Attendee Registration fields saved. Make additional changes or %1$s', 'event-tickets' ), $link );
```